docs/Testing/ai-foundry-integration.md

```diff
@@ -1,6 +1,9 @@
 # Deploying SQL MCP Server implemented in Data API builder and Integrating with Azure AI Foundry
 
-This document provides an end‑to‑end guide to stand up a **SQL MCP Server** with **Model Context Protocol (MCP)** tools implemented in **Data API builder (DAB)** container that also exposes **REST** and **GraphQL** endpoints, and to integrate those MCP tools with an **Azure AI Foundry Agent**. 
+This document provides an end‑to‑end guide to stand up a **SQL MCP Server** with **Model Context Protocol (MCP)** tools implemented in **Data API builder (DAB)** container that also exposes **REST** and **GraphQL** endpoints, and to integrate those MCP tools with an **Azure AI Foundry Agent**.
+
+<img width="2398" height="2178" alt="image" src="https://github.com/user-attachments/assets/9ba6877a-e5d9-4068-862f-36d988370c67" />
```

Comment:
shouldnt we checkin the image under the media folder, and include that file here instead of providing the github url?

Reply:
this is the media folder: https://github.com/Azure/data-api-builder/tree/725bf8c12b8488525b7d8464d9f662ea297e60dd/docs/media